tdex-run-mac

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```

Comment:
is it this to be checked in? If yes lets move to scripts folder

Reply:
Nice catch, it has been pushed by mistake